data_tools/build_rawframes.py

```diff
@@ -104,7 +104,7 @@ def parse_args():
                         help='resize image width')
     parser.add_argument("--new_height", type=int,
                         default=0, help='resize image height')
-    parser.add_argument("--num_gpu", type=int, default=8, help='number of GPU')
+    parser.add_argument("--num_gpu", type=int, default=1, help='number of GPU')
```

Comment:
Please do not modify this number.

Reply:
I have some questions about this parameter? Why this parameter could be 8?

Reply:
The training parameters were set assuming 8 GPUs are used. So we let the default number to be 8 to avoid suprises. It's better keep it 8.

Reply:
Okay. I just edit to the default value. :)